### ISSUE TITLE
New publish Wizard need better testing.

### ISSUE THREAD

**Issue by manonthegithub** [dbpedia/databus#66]
* List of files is not properly showing the files in at least safari...
<img width="831" alt="image" src="https://github.com/dbpedia/databus/assets/10817016/896a5a23-f64d-4696-9675-2d6eed3d7d63">
though the version metadata below looks good

* adding wrong Uris for example google.com makes the interface hang on the add file action
<img width="935" alt="image" src="https://github.com/dbpedia/databus/assets/10817016/abff3bd6-3962-46aa-b319-3a57f18337b1">

* The publish wizard seems to have now too much text... the hidden unless hover hints were better I think -> here probably is a matter of taste

* Attribution and Derived from fields a bit not clear what we want there? links text how should it look like? Maybe some example text in the text field?

**Comment by kurzum** [comment]
@manonthegithub  re-assess

**Comment by manonthegithub** [comment]
* Attribution and Derived from fields a bit not clear what we want there? links text how should it look like? Maybe some example text in the text field? -> still there
* not sure is that is as expected, when I save a single file, after publish it is renamed to version (<version>.md)

**Comment by manonthegithub** [comment]
I think in general publish wizard need more work on usability, UX

**Comment by manonthegithub** [comment]
Ref #61

**Comment by holycrab13** [comment]
> The publish wizard seems to have now too much text... the hidden unless hover hints were better I think -> here probably is a matter of taste 

Agree, it is a matter of preference. I had a look at how github does it in their interface when there are a lot of complex fields (e.g. in the user settings) and took inspiration from there.

**Comment by manonthegithub** [comment]
Pretty often it doesn’t need too much text, but better more precise and understandable formulations (for the person who sees it first time -> those who use it regularly don’t need explanations)
